apps/storybook/src/GroupProfileAbout.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { GroupProfileAbout } from '@asap-hub/react-components';
+import { createGroupResponse } from '@asap-hub/fixtures';
+import { boolean } from '@storybook/addon-knobs';
+
+export default {
+  title: 'Templates / Group Profile / About',
+  component: GroupProfileAbout,
+};
+
+export const Normal = () => (
+  <GroupProfileAbout
+    {...createGroupResponse()}
+    leaders={
+      boolean('Has Leaders', true)
+        ? createGroupResponse().leaders.map(({ user, role }) => ({
+            user: {
+              ...user,
+              teams: user.teams.map((team) => ({ ...team, href: '#' })),
+            },
+            role,
+            href: '#',
+            teams: user.teams.map((team) => ({ ...team, href: '#' })),
+          }))
+        : []
+    }
+    teams={
+      boolean('Has Teams', true)
```

Comment:
can

Reply:
Oh, yeah, I remember actually having thought about that too (who knows if the fixture doesn't just have `[]` for that), but then didn't do anything about it. Adding `teamsCount` etc...